doc/supported_operations.md

```diff
@@ -0,0 +1,16 @@
+# Containerized Data Importer supported operations
+The Containerized Data Importer (CDI) supports importing data/disk images from a variety of sources, and it can be hard to understand which combination of operations is supported and if that operation requires scratch space. This document explains the combinations that are supported and which ones require scratch space. All Kubevirt content type operations will support both file system Data Volumes (DV) as well as block volume DVs.
+
+## Supported matrix
+
+| Content-Type | http | https | http basic auth | Registry | Upload | S3 Bucket  |
```

Comment:
To the untrained eye it's not clear if content-type refers to the x or y axis.  Can you use shading or some other technique to better associate that heading with the entries below it?.

Maybe upload should go on the end since the rest are import operations?

Reply:
Done